eng/versioning/set_versions.py

```diff
@@ -221,6 +224,20 @@ def prep_version_file_for_source_testing(build_type):
                 if hasattr(module, 'current') and not module.current == module.dependency:
                     module.dependency = module.current
                     file_changed = True
+                # In order to ensure that the From Source runs are effectively, ensure that
```

Comment:
I think there might be a word missing after "effectively" here.

Reply:
I just pushed a new commit that turns the sentence fragment into an actual sentence.